spec/features/create_an_auth_token_spec.rb

```diff
@@ -39,13 +39,13 @@ def notify_email(subscriber, destination, redirect)
         "body" => hash_including(
           "email_address" => subscriber.address,
           "personalisation" => hash_including(
-            "subject" => "Confirm your email address",
+            "subject" => "Manage your GOV.UK email subscriptions",
             "body" => include("http://www.dev.gov.uk#{destination}?token="),
           ),
         ),
       )
       .with { |request|
-        token = request.body.match(/token=([^&)]+)/)[1]
+        token = request.body.match(/token=([^&\\]+)/)[1]
```

Comment:
I'm not really clear why this regex isn't matching [this line](https://github.com/alphagov/email-alert-api/pull/1175/files#diff-fc85cafa53fc05719fa7019655ddc05fR38). However it's aimed at stripping the token off of the URL. We've changed it from some markdown to something that fits onto a newline.

By dropping a debugger here and taking the `request.body` into rubular. I got the following to work.